quizbizz/src/util/api/fetchApi.js

```diff
@@ -0,0 +1,31 @@
+import Axios from "axios";
+
+const fetchApi = async (url, body, isAuthRequired) => {
+  const token = localStorage.getItem("token");
+  try {
+    const response = await Axios.get(
+      url,
+      { body },
+      {
+        headers: {
+          authorization: isAuthRequired ? token : "",
+        },
+      }
+    );
+
+    return {
+      data: response.data,
+      success: true,
+      message: "Api Request Successfull",
+    };
+  } catch (e) {
+    console.log(e);
```

Comment:
suggestion: use console.error for displaying errors

Reply:
Done